spec/features/admin_can_add_a_book_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
```

Comment:
This test is solid. I would, however, like to see some kind of proof that a User can NOT add a book.